app/main.py

```diff
@@ -1 +1,42 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
```

Comment:
```suggestion
    def __init__(
          self, 
          name: str,
          appetite: int,
          is_hungry: bool = True
    ) -> None:
```
Please be patient with indentation. This is the right way to split line